readme_template.md

```diff
@@ -0,0 +1,83 @@
+
+Describe what the application(s)/services(s)/content of this repository are and how they are used in relation to other applications and services.
+
+For large README files a table of content can be added, which can be automated with the [Auto Markdown TOC extension](https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc) (adjust the depth to start from with `depthFrom:2` after the `TOC` marker):
+
+<!-- TOC depthfrom:2 -->
+
+- [:blue_book: Usage](#blue_book-usage)
+- [:whale: or :arrow_forward: Run a release](#whale-or-arrow_forward-run-a-release)
+- [:hammer_and_wrench: Prerequisites](#hammer_and_wrench-prerequisites)
+- [:desktop_computer: Development](#desktop_computer-development)
+    - [Getting Started](#getting-started)
+    - [Building](#building)
+    - [Testing](#testing)
+- [:art: Assets or Sprites](#art-assets-or-sprites)
+- [:handshake: Contributing](#handshake-contributing)
+- [:rocket: Production](#rocket-production)
+- [:mag: Links to further information](#mag-links-to-further-information)
+
+<!-- /TOC -->
+
+## :blue_book: Usage
+
+Describe or link all applications/services in this repository as well as their command line options or environment variables.
+
+## :whale: or :arrow_forward: Run a release
+
+If running a release is the easiest way to run the application or services of the repository, this section should be the first one.
+
+Describe where the release can be downloaded (`docker pull` or release download) and how to start it locally. This must include all required environment variables. Use the :whale: icon for docker containers and the :arrow_forward: for all other releases.
+
+If there is a lot of configuration, provide the command to start the service with our scripts in [docker-images](https://github.com/skillslab/docker-images) first, otherwise after the [Docker](https://www.docker.com/) or [Docker Compose](https://docs.docker.com/compose/) command.
+
+## :hammer_and_wrench: Prerequisites
+
+You have to add clone instructions if a simple pull is not sufficient. For example if submodules have to be checked out.
+
+Add instructions for software that has to be installed on the machine to run the code or link to another repository or other documentation which covers the setup. Often used examples are [ansible repo](https://github.com/skillslab/ansible) and [Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers).
+
+## :desktop_computer: Development
+
+### Getting Started
+
+Describe how to start up the development environment. Either with docker, Visual Studio Code and Dev Containers or the Unreal Engine. If there are multiple ways, mention all, but start with the most used. Also mention the differences.
+
+### Building
+
+In this section describe how the project can be built and describe which configurations are available.
+
+### Testing
+
+Explain where the tests are organized and how they can be executed.
+
+## :art: Assets or Sprites
```

Comment:
Could this be a part of prerequisites?

Reply:
Yes, I meant the assets for the UI, because without them, you won't see any images. So, it's more of a prerequisite before you start developing.